services/src/main/java/org/keycloak/services/clientpolicy/executor/HolderOfKeyEnforceExecutor.java

```diff
@@ -28,6 +28,10 @@
 import org.keycloak.representations.RefreshToken;
 import org.keycloak.representations.idm.ClientRepresentation;
 import org.keycloak.services.clientpolicy.*;
+import org.keycloak.services.clientpolicy.context.LogoutRequestContext;
```

Comment:
I suggest to remove star import. In general, we try to avoid star imports in Keycloak. Star import was not introduced in this PR, but maybe can be done as part of this PR when you are on this?

Reply:
@mposolda I see. I've recognized that keycloak does not use star import in its coding style but I've forgot it. I'll replace this star import with explicit import directives.